Arion/include/Arion/ExpandingPolytopeAlgorithm.hpp

```diff
@@ -151,15 +165,15 @@ ContactManifold CalculateContactManifold(
     do
     {
         //Get distance and direction to the polytope's face that is nearest to the origin
-        ConvexHull::Faces const& chFaces = convexHull.GetFaces();
-        face = std::min_element(chFaces.begin(), chFaces.end(),
-            [](ConvexHull::Face const& a, ConvexHull::Face const& b) -> bool
-        {
-            return a.GetHyperPlane().GetDistance() < b.GetHyperPlane().GetDistance();
+        planeIt = std::min_element(convexHull.planes.begin(), convexHull.planes.end(),
+                [](auto& a, auto& b) -> bool {
+                    return a.GetDistance() < b.GetDistance();
         });
-        direction = face->GetHyperPlane().GetNormal();
+        faceIndex = std::distance(convexHull.planes.cbegin(), planeIt);
+        direction = planeIt->GetNormal();
         assert(!glm::isnan(direction.x));
-        closestFaceDistance = face->GetHyperPlane().Distance({0, 0, 0});
+        //FixMe: Replace with glm::abs(hyperPlane.GetDistance())
```

Comment:
why not now?

Reply:
@dsodintsov I think I would need a task for it, since I wont to fix the stable revision and than work with performance